src/libs/Notification/PushNotification/index.native.js

```diff
@@ -48,6 +48,30 @@ function pushNotificationEventCallback(eventType, notification) {
     action(payload);
 }
 
+/**
+ * Register push notification callbacks. This is separate from namedUser registration because it needs to be executed
```

Comment:
NAB, this code can still be moved and break stuff in not obvious ways so I would personally leave a warning about that here and use angry all caps, but up to you. 😂

Reply:
done